### DISCUSSION TITLE
AppImage build failing as zlib and _struct are not getting created while build is performed

### DISCUSSION THREAD

**Discussion by swetayadav1** [pyinstaller/pyinstaller#7049] [PyInstaller]
```
echo ">>> Building the release package"
${PYTHON_VENV} -m PyInstaller ndrive.spec --clean --noconfirm
```

AppImage build failing as zlib and _struct are not getting created while build is performed

<img width="877" alt="image" src="https://user-images.githubusercontent.com/106366788/187141464-5d2fc629-c407-4692-be44-1284a644ef69.png">

Any suggestion how to fix this issue?

Python version: 3.9.5
Pyinstaller: 5.0

**Comment by rokm** [comment]
There should not be a `_struct` and a `zlib`  *folder*. There used to be with older versions of PyInstaller, but that was due to a bug (see #5851). 

So you need to update your validation script...
